editor/plugins/tiles/tile_map_editor.cpp

```diff
@@ -136,9 +140,15 @@ void TileMapEditorTilesPlugin::_update_tile_set_sources_list() {
 		return;
 	}
 
-	for (int i = 0; i < tile_set->get_source_count(); i++) {
-		int source_id = tile_set->get_source_id(i);
+	int prev_source = -1;
```

Comment:
I am nitpicking, but reusing old_source might have been simpler there. ^^
Something like:
```
if (!tile_set->has_source(old_source)) {
    old_source = -1;
}
```
And re-use `old_source` in place of `prev_source` later on.

Reply:
Makes sense.